agent-inject/agent/annotations.go

```diff
@@ -111,6 +111,15 @@ const (
 	// and gid) is set on the injected Vault Agent containers
 	AnnotationAgentSetSecurityContext = "vault.hashicorp.com/agent-set-security-context"
 
+	// AnnotationAgentServiceAccountTokenVolumeName is the optional name of a volume containing a
+	// service account token
+	AnnotationAgentServiceAccountTokenVolumeName = "vault.hashicorp.com/agent-service-account-token-volume-name"
+
+	// AnnotationAgentServiceAccountTokenVolumePath is the optional path to a service account token
+	// within a volume named by "vault.hashicorp.com/agent-service-account-token-volume-name" or
+	// discovered by searching for service account volume mounts - the default is "token"
+	AnnotationAgentServiceAccountTokenVolumePath = "vault.hashicorp.com/agent-service-account-token-volume-path"
+
```

Comment:
Anyway we can make these annotations shorter?  They're very close to max length.

Reply:
@jasonodonnell any update on this? personally I don't see why we want to shorten the names as they are descriptive for a reason and they do fit within the limits, but if you have any other suggestions for names I am happy to change them